src/features/Auth/SignInForm/SignInForm.module.scss

```
@@ -0,0 +1,22 @@
+@use '@/styles/variables.scss' as *;
+
+.form {
+  width: 100%;
+}
+
+.loginBtn {
+  width: 100%;
+  margin-top: 2rem;
+}
+
+.kakaoLoginBtn {
+  width: 100%;
+  margin-top: 0.5rem;
+  margin-bottom: 2rem;
+  border: 0;
+}
```

다른 페이지에서는 `margin-bottom`이 `1rem` 이더라구용
```suggestion
.kakaoLoginBtn {
  width: 100%;
  margin-top: 0.5rem;
  margin-bottom: 1rem;
  border: 0;
}
```